openspec/changes/stabilize-local-ci-postgres-tests/spec.md

```
@@ -0,0 +1,99 @@
+
+#### Scenario: `just backend-test-fast` passes
+
+- GIVEN the helper is configured and the Justfile step is updated
+- WHEN `just backend-test-fast` runs
+- THEN it MUST exit zero
+
+#### Scenario: `just backend-test-postgres` runs and outcome is recorded
+
+- GIVEN the documentation is reconciled
+- WHEN `just backend-test-postgres` runs
+- THEN the command MUST execute and the exact command, exit code, elapsed time, and outcome MUST be in `verify-report.md`
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)